spotinst_sdk2/clients/ocean_cd/__init__.py

```diff
@@ -0,0 +1,950 @@
+
+        excluded_node_update_dict = self.exclude_missing(
+            json.loads(request.toJSON()))
+
+        formatted_node_update_dict = self.convert_json(
+            excluded_node_update_dict, self.underscore_to_camel)
+
+        body_json = json.dumps(formatted_node_update_dict)
+
+        response = self.send_post(
+            body=body_json,
+            url=self.__base_oceancd_vp_url,
+            entity_name=self.ENTITY_NAME2)
+
+        formatted_response = self.convert_json(
+            response, self.camel_to_underscore)
+
+        return formatted_response["response"]["items"]
+
+    def get_all_oceancd_verification_provider(self):
+        """
+        List all Ocean CD verification providers.
+
+        # Returns
+        (List): List of Ocean CD Verification Provider API response
+        """
+        response = self.send_get(
+            url=self.__base_oceancd_vp_url,
+            entity_name=self.ENTITY_NAME2)
+
+        formatted_response = self.convert_json(
+            response, self.camel_to_underscore)
+
+        return formatted_response["response"]["items"]
+
+    def get_oceancd_verification_provider(self, name: str):
+        """
+        Get an existing  Ocean CD Verification Provider.
+
+        # Arguments
+        name (String): The identifier of the Ocean CD Verification Provider
+
+        # Returns
+        (Object): OceanCD Verification Provider API response
+        """
+        response = self.send_get(
+            url=self.__base_oceancd_vp_url + "/" + name,
+            entity_name=self.ENTITY_NAME2)
+
+        formatted_response = self.convert_json(
+            response, self.camel_to_underscore)
+
+        return formatted_response["response"]["items"][0]
+
+    def update_oceancd_verifcation_provider(self, provider_update: oceancd_vp.VerificationProvider, name: str):
+        """
+        Full Update of Ocean CD Verification Provider configuration. All non included fields will be nullified
+
+        # Arguments
+        name (String): The identifier name of the Ocean CD Verification Provider
+        provider_update (VerificationProvider): VerificationProvider Object
+
+        # Returns
+        (Boolean): Response Status
+        """
+        request = oceancd_vp.CreateVerificationProviderRequest(provider_update)
+
+        excluded_node_update_dict = self.exclude_missing(
+            json.loads(request.toJSON()))
+
+        formatted_node_update_dict = self.convert_json(
+            excluded_node_update_dict, self.underscore_to_camel)
+
+        body_json = json.dumps(formatted_node_update_dict)
+
+        response = self.send_put(
+            body=body_json,
+            url=self.__base_oceancd_vp_url + "/" + name,
+            entity_name=self.ENTITY_NAME2)
+
+        formatted_response = self.convert_json(
+            response, self.camel_to_underscore)
+
+        return formatted_response["response"]["status"]
```

Comment:
Shouldn't return only status

Reply:
We are only getting status as response for this api